build/FrameworkPackage/MakeFrameworkPackage.ps1

```diff
@@ -74,40 +65,35 @@ ForEach ($input in ($inputs -split ";"))
     Write-Verbose "Copying $inputBasePath\Themes"
     Copy-IntoNewDirectory -IfExists $inputBasePath\Themes $fullOutputPath\PackageContents\Microsoft.UI.Xaml
 
+    $sdkReferencesPath=$kitsRoot10+"\References\10.0.18323.0";
```

Comment:
>18323 [](start = 53, length = 5)

Can you parse this out of SdkVersion.props somehow? Maybe just grab the large version out of the SDKVersion* property group?